include/zim/zim.h

```diff
@@ -76,6 +78,7 @@ namespace zim
     COUNT
   };
 
+  std::vector<std::pair<std::string, std::string>> getVersions();
```

Comment:
Please `typedef` `std::vector<std::pair<std::string, std::string>>` as `LibVersions` (or something similar) and use that shorter name instead.

Reply:
OK, see next fixup commit.